openquake/calculators/base.py

```diff
@@ -505,20 +512,22 @@ def read_gmfs_from_csv(calc):
     :param calc: a ScenarioDamage or ScenarioRisk calculator
```

Comment:
It would be better to rename this function from `read_gmfs_from_csv` to `read_gmfs_from_file`, now that the gmfs can also be read from NRML files.